packages/devops/scripts/ci/validateBranchName.sh

```diff
@@ -60,7 +60,6 @@ function validate_name_chars() {
 }
 
 branch_name=$(get_branch_name)
-check_name_is_not_reserved $branch_name
```

Comment:
Why this change?

Reply:
Fixes an existing (non breaking issue) - the function `check_name_is_not_reserved ` was removed, but its call was not.

This function used to ensure that the branch was not `e2e`, now we maintain that branch (clone of `master`)

Reply:
Gotcha